pkg/controller/migmigration/task.go

```diff
@@ -30,6 +30,7 @@ const (
 	ResticRestarted               = "ResticRestarted"
 	QuiesceApplications           = "QuiesceApplications"
 	EnsureQuiesced                = "EnsureQuiesced"
+	ReactivateApplications        = "ReactivateApplications"
```

Comment:
Maybe this should be UnQuiesceApplications for consistency?

Reply:
Thanks! Missed this one